mappings/net/minecraft/world/gen/treedecorator/AlterGroundTreeDecorator.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4658 net/minecraft/world/gen/tree/AlterGroundTreeDecorator
+CLASS net/minecraft/class_4658 net/minecraft/world/gen/treedecorator/AlterGroundTreeDecorator
```

Comment:
Oof it hurts my eyes, but it's technically more consistent

Reply:
Yeah, it's not the most elegant solution but keeping them in the "tree" namespace was more confusing, at least to me

Reply:
Is it possible to put it in like `decorator.tree` or is this actually unrelated to decorators

Reply:
They are unrelated to normal decorators. Normal decorators get positions for features to generate, but tree decorators add vines/cocoa/podzol/extra stuff to trees

Reply:
Should they even be called `decorator`s then? Since feature placement decorators and what this appears to be seem to be two entirely different things

Reply:
Think this name comes from mojang registry